src/clj/witan/send/model/data_products.clj

```diff
@@ -0,0 +1,193 @@
+(ns witan.send.model.data-products
```

Comment:
why data-products?

Reply:
data products are the things we make from the main projection. I've not found a better name (happy to have one).